### PR TITLE
Allow the translation bot to recompile the htmls

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
Do not exit if `composer install` was not executed and there is a system variable `TRANSLATION_BOT` with a value `YES`

Basically, this allows the translation bot to create the error pages by skipping the composer part

### Documentation Changes Required

@wilsonge a simple review should be enough here, it solves the issue https://github.com/joomla/joomla-cms/pull/35379#issuecomment-917367450

### CODE REVIEW

**Comment by HLeithner** [comment]
looks odd can we add parameter something like --no-vendor-cleanup ? so it's more generic?

File: build/build-modules-js/init/cleanup-media.es6.js [hunk 1]
```diff
@@ -12,6 +12,12 @@ const RootPath = process.cwd();
  * @returns {Promise}
  */
 module.exports.cleanVendors = async () => {
+  if (process.env.TRANSLATION_BOT === 'YES') {
+    // eslint-disable-next-line no-console
+    console.log('Cleanup the Vendor skipped');
```
Comment (brianteeman) [inline build/build-modules-js/init/cleanup-media.es6.js]:
```suggestion
    console.log('Cleanup the Vendor folder skipped');
```

**Comment by dgrammatiko** [comment]
> looks odd can we add parameter something like --no-vendor-cleanup ? so it's more generic?

ENV vars can be used for the same result. If we go with the `--no-vendor-cleanup` option in the script we have to add one more procedure in the build.js, basically the ENV way is shorter. That said if you insist I can refactor the build.js to accept `--no-vendor-cleanup` and do the same

**Comment by HLeithner** [comment]
ok then ENV but please change the name ;-)

**Comment by dgrammatiko** [comment]
> ok then ENV but please change the name ;-)

Give me an alternative, I'm not good at naming things

How about `SKIP_COMPOSER_CHECK`?

**Comment by HLeithner** [comment]
Thanks

**Comment by HLeithner** [comment]
@dgrammatiko seems not to work https://github.com/joomla-translation-bot/joomla-cms/runs/3575595533?check_suite_focus=true

**Comment by dgrammatiko** [comment]
@HLeithner https://github.com/joomla-translation-bot/joomla-cms/pull/4

the value is case sensitive

**Pull request merged**